cpp/ql/lib/experimental/semmle/code/cpp/ir/dataflow/internal/DataFlowImpl.qll

```diff
@@ -1169,8 +1169,21 @@ pragma[nomagic]
 private int branch(NodeEx n1, Configuration conf) {
   result =
     strictcount(NodeEx n |
-      flowOutOfCallNodeCand1(_, n1, _, n, conf) or flowIntoCallNodeCand1(_, n1, n, conf)
-    )
+        flowOutOfCallNodeCand1(_, n1, _, n, conf) or flowIntoCallNodeCand1(_, n1, n, conf)
+      ) + sum(ParamNodeEx p1 | | getLanguageSpecificFlowIntoCallNodeCand1(n1, p1, conf))
+}
+
+/**
+ * Gets an additional term that is added to `branch` and `join` when deciding whether
+ * the amount of forward or backward branching is within the limit specified by the
+ * configuration.
+ */
+pragma[nomagic]
+private int getLanguageSpecificFlowIntoCallNodeCand1(
+  ArgNodeEx n1, ParamNodeEx p1, Configuration conf
+) {
+  flowIntoCallNodeCand1(_, n1, p1, conf) and
+  result = getAdditionalFlowIntoCallNodeTerm(p1.projectToNode(), n1.projectToNode())
```

Comment:
This line also needs a switch.

Reply:
Oh, oops. Thanks!

Reply:
Fixed in 6d77696043db0e64e18c06022de2467320cdce74. (I also renamed the columns to match the code around it better.)